### ISSUE TITLE
Effectively Installed

### ISSUE THREAD

**Issue by iMattPro** [PayBas/BreadcrumbMenu#1]
You do not need effectively installed in your migration. That is only needed, really, when performing migrations that could involve older pre-3.1/pre-extension data. Since, as far as I can tell, this extension is brand new, and the config var `breadcrumbmenu_version` has never existed before, and therefor you're not updating from some older previous installation prior to migrations (a MOD), effectively_installed() is not needed.

The depends_on() method in migrations is used to ensure migrations run in a particular order or sequence. This is what should be used in future migration files to make sure the correct database changes get made in the correct order.

The effectively_installed() method is used to prevent migrations from writing data that may already exist from a previous database change made prior to the migrations system in 3.1 (MODs).

So in a nutshell, effectively_installed() is only needed if your extension was once a MOD that used UMIL to change the database :8ball: Check out the phpBB Board Rules migration files, for example: https://github.com/phpbb-extensions/boardrules/tree/master/migrations/v10x
